data/human/hails.txt

```diff
@@ -2538,7 +2538,7 @@ phrase "friendly pirate"
 		" You'll find some on the nearest pirate planet."
 		" You'll find some on the nearest anarchist planet."
 		" You'll find some on the nearest planet not oppressed by the man."
-		" You'll find some where ever the man does not go."
+		" You'll find somewhere ever the man does not go."
```

Comment:
"some" actually refers to "contraband".

Reply:
Thanks, I didn't notice that earlier. Should it then become "You'll find some wherever the man does not go."?

Reply:
Yes, "wherever" is better. I wasn't sure about it, but in this case "where ever" doesn't fit.